examples/cpu_monitor.rs

```diff
@@ -1,10 +1,8 @@
-extern crate docopt;
-extern crate rayon;
 #[macro_use]
 extern crate serde_derive;
-extern crate serde;
 
 use docopt::Docopt;
+use rayon;
```

Comment:
I know rustfix does this, but I think this `use` is not actually needed, as `rayon` should be in the crate prelude. There are a few cases of this, but I won't comment them all. Not a _big_ deal, just something that could be cleaned up.